---
layout: paper
title: "Explaining complex codon usage patterns with selection for translational efficiency, mutation bias, and genetic drift."
year: "2011"
ref: "Shah & Gilchrist <i>PNAS</i> 2011"
nickname: mutsel-efficiency
journal: "PNAS"
volume: 108
issue: 25
pages: 10231-10236
authors: "Shah P and Gilchrist MA."
image: /images/papers/shah-cub-treff.jp2
redirect_from: 
fulltext: 
pdf: /pdfs/papers/Shah_PNAS_2011.pdf
pdflink: 
github: 
pmid: 21646514
pmcid: PMC3121864
f1000: 
doi: 10.1073/pnas.1016719108
dryad_doi: 
altmetric_id: 
figshare: 
category: paper
published: true
embargo: false
peerreview: true
review: false
tags: mutation bias translation efficiency CUB popgen
---
# Abstract 

The genetic code is redundant with most amino acids using multiple codons. In many organisms, codon usage is biased toward particular codons. Understanding the adaptive and nonadaptive forces driving the evolution of codon usage bias (CUB) has been an area of intense focus and debate in the fields of molecular and evolutionary biology. However, their relative importance in shaping genomic patterns of CUB remains unsolved. Using a nested model of protein translation and population genetics, we show that observed gene level variation of CUB in Saccharomyces cerevisiae can be explained almost entirely by selection for efficient ribosomal usage, genetic drift, and biased mutation. The correlation between observed codon counts within individual genes and our model predictions is 0.96. Although a variety of factors shape patterns of CUB at the level of individual sites within genes, our results suggest that selection for efficient ribosome usage is a central force in shaping codon usage at the genomic scale. In addition, our model allows direct estimation of codon-specific mutation rates and elongation times and can be readily applied to any organism with high-throughput expression datasets. More generally, we have developed a natural framework for integrating models of molecular processes to population genetics models to quantitatively estimate parameters underlying fundamental biological processes, such a protein translation.
