---
layout: paper
title: "Improved ribosome-footprint and mRNA measurements provide insights into dynamics and regulation of yeast translation."
year: "2016"
ref: Weinberg & Shah <i>et al. Cell Rep</i> 2016
nickname: improved-riboprofile
journal: "Cell Rep"
volume: 14
issue: 7
pages: 1787-1799
authors: "Weinberg DE*, Shah P*, Eichhorn SW, Hussmann JA, Plotkin JB, and Bartel DP."
image: /images/papers/weinberg-translation.png
redirect_from: 
fulltext: 
pdf: /pdfs/papers/Weinberg_CellRep_16.pdf
supplement: /pdfs/papers/Weinberg_CellRep_16_s.pdf
pdflink: 
github: https://github.com/shahpr/SMoPT
github_n: SMoPT
pmid: 26876183
pmcid: PMC4767672
f1000: 
doi: 10.1016/j.celrep.2016.01.043
dryad_doi: 
altmetric_id: 
figshare: 
category: paper
published: true
embargo: false
peerreview: true
review: false
tags: riboprofiling CHX mRNA protein translation yeast SMoPT
---
# Abstract 

Ribosome-footprint profiling provides genome-wide snapshots of translation, but technical challenges can confound its analysis. Here, we use improved methods to obtain ribosome-footprint profiles and mRNA abundances that more faithfully reflect gene expression in <i>Saccharomyces cerevisiae</i>. Our results support proposals that both the beginning of coding regions and codons matching rare tRNAs are more slowly translated. They also indicate that emergent polypeptides with as few as three basic residues within a 10-residue window tend to slow translation. With the improved mRNA measurements, the variation attributable to translational control in exponentially growing yeast was less than previously reported, and most of this variation could be predicted with a simple model that considered mRNA abundance, upstream open reading frames, cap-proximal structure and nucleotide composition, and lengths of the coding and 5â€™-untranslated regions. Collectively, our results reveal key features of translational control in yeast and provide a framework for executing and interpreting ribosomeprofiling studies.
