---
layout: paper
title: "The landscape of transcriptional and translational changes over 22 years of bacterial adaptation."
year: "2021"
ref: "Favate <i>et al. bioRxiv</i> 2021"
nickname: ltee_riboseq
journal: "biRxiv"
volume: 
issue: 
pages: 
authors: "Favate JS, Liang S, Yadavalli SS, and Shah P."
lab: "Shah"
image: /images/papers/favate-ltee.png
redirect_from: 
fulltext: https://www.biorxiv.org/content/10.1101/2021.01.12.426406v1
pdf: /pdfs/papers/Favate_bioRxiv_2021.pdf
pdflink: 
github: https://github.com/shahlab/LTEE-gene-expression
github_n:
pmid: 
pmcid: 
f1000: 
doi: 10.1101/2021.01.12.426406
dryad_doi: 
altmetric_id: 
figshare: 
category: paper
published: false
embargo: false
peerreview: false
review: false
tags: ecoli riboseq ltee 
---
# Abstract 

Organisms can adapt to an environment by taking multiple mutational paths. This redundancy at the genetic level, where many mutations have similar phenotypic and fitness effects, can make untangling the molecular mechanisms of complex adaptations difficult. Here we use the E. coli long-term evolution experiment (LTEE) as a model to address this challenge. To bridge the gap between disparate genomic changes and parallel fitness gains, we characterize the landscape of transcriptional and translational changes across 11 replicate populations evolving in parallel for 50,000 generations. By quantifying absolute changes in mRNA abundances, we show that not only do all evolved lines have more mRNAs but that this increase in mRNA abundance scales with cell size. We also find that despite few shared mutations at the genetic level, clones from replicate populations in the LTEE are remarkably similar to each other in their gene expression patterns at both the transcriptional and translational levels. Furthermore, we show that the bulk of the expression changes are due to changes at the transcriptional level with very few translational changes. Finally, we show how mutations in transcriptional regulators lead to consistent and parallel changes in the expression levels of downstream genes, thereby linking genomic changes to parallel fitness gains in the LTEE. These results deepen our understanding of the molecular mechanisms underlying complex adaptations and provide insights into the repeatability of evolution.

